github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-presubmits.yaml

```diff
@@ -414,3 +414,36 @@ presubmits:
       - name: kubevirtci-fossa
         secret:
           secretName: kubevirtci-fossa-token
+  - name: pull-containerized-data-importer-non-csi-k8s-local
```

Comment:
Should we discuss the benefits of local vs old (non csi) hpp?  I don't have much of a preference.  But hpp is nice because it's a real dynamic provisioner.  With local, could maybe run out of volumes?

Reply:
I'll use classic hpp since it seemed to be the favorite among the team.